2nd.py

```
@@ -0,0 +1,11 @@
+# simple_utils.py - A tiny utility library
+
+def reverse_string(text):
+    """Reverses the characters in a string."""
+    return text[::-1]
+
+def count_words(sentence):
+    return len(sentence.split())
+
+def celsius_to_fahrenheit(celsius):
+    return (celsius * 9/5) + 32
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)